cli/command/context/show.go

```diff
@@ -25,10 +25,10 @@ func newShowCommand(dockerCli command.Cli) *cobra.Command {
 
 func runShow(dockerCli command.Cli) error {
 	context := dockerCli.CurrentContext()
-	metadata, err := dockerCli.ContextStore().GetMetadata(context)
+	fmt.Fprintln(dockerCli.Out(), context)
+	_, err := dockerCli.ContextStore().GetMetadata(context)
 	if err != nil {
 		return err
 	}
-	fmt.Fprintln(dockerCli.Out(), metadata.Name)
 	return nil
 }
```

Comment:
For this one, I'm actually wondering if it would make sense to "just show" what's currently configured as context, and skip the validation altogether. As `docker show` could be commonly used to set the prompt, there were some concerns about performance here; https://github.com/docker/cli/issues/2584#issuecomment-1222720408